tests/transfer.test.ts

```
@@ -0,0 +1,170 @@
+      createTransfer({ senderUserId: SENDER_ID, to: "@bob", amountAcbu: "10" })
+    ).rejects.toThrow("Sender user not found");
+  });
+
+  it("rejects unverified sender (KYC)", async () => {
+    (mockUser.findUnique as jest.Mock).mockResolvedValue({
+      stellarAddress: SENDER_STELLAR,
+      kycStatus: "pending",
+    });
+
+    await expect(
+      createTransfer({ senderUserId: SENDER_ID, to: "@bob", amountAcbu: "10" })
```

ESLint

[error] Insert `,`

(prettier/prettier)